File: datasets_v2/education/hackathon/compose/about_us.kt
import androidx.compose.ui.tooling.preview.Preview
import androidx.compose.foundation.background
import androidx.compose.foundation.layout.*
import androidx.compose.foundation.lazy.LazyColumn
import androidx.compose.foundation.lazy.items
import androidx.compose.material3.*
import androidx.compose.runtime.*
import androidx.compose.ui.Alignment
import androidx.compose.ui.Modifier
import androidx.compose.ui.draw.paint
import androidx.compose.ui.graphics.Color
import androidx.compose.foundation.Image
import androidx.compose.ui.res.painterResource
import androidx.compose.ui.layout.ContentScale
import androidx.compose.ui.text.style.TextAlign
import androidx.compose.ui.unit.sp
import androidx.compose.ui.unit.dp

@Composable
fun about_us() {
  Column(modifier = Modifier.paint(painterResource(id = R.drawable.rsz_home_bg), contentScale = ContentScale.Crop).fillMaxWidth().fillMaxHeight(), horizontalAlignment = Alignment.CenterHorizontally) {
    Text(text = "About Us", modifier = Modifier.wrapContentWidth().wrapContentHeight(), color = Color(0xFFFFFFFF), fontSize = 50.sp, textAlign = TextAlign.Center)
    Image(painter = painterResource(id = R.drawable.rsz_ow), contentDescription = null, modifier = Modifier.background(Color.Unspecified).size(200.0.dp, 200.0.dp).padding(top = 4.0.dp), contentScale = ContentScale.Fit)
    Image(painter = painterResource(id = R.drawable.sah), contentDescription = null, modifier = Modifier.background(Color.Unspecified).size(400.0.dp, 100.0.dp).padding(top = 4.0.dp), contentScale = ContentScale.Fit)
    Box(modifier = Modifier.fillMaxWidth(), contentAlignment = Alignment.TopEnd) {
    Text(text = "v 0.0.1", modifier = Modifier.wrapContentHeight().width(372.0.dp), color = Color(0xFFF08080), textAlign = TextAlign.Center)
    }
    Text(text = "Made with ♥ for India, in India. ", modifier = Modifier.wrapContentWidth().wrapContentHeight().padding(top = 20.0.dp), color = Color(0xFFFFFFFF), fontSize = 20.sp)
    Text(text = "http://neophytes.github.io/microsoft-pragyan-hackathon/", modifier = Modifier.wrapContentWidth().wrapContentHeight().padding(top = 32.0.dp))
  }
}


@Preview(showBackground = true)
@Composable
fun about_us_preview() {
    MyApplicationTheme {
        about_us()
    }
}

File: datasets_v2/education/hackathon/swiftui/About_us.swift
import SwiftUI

struct About_us: View {
  var body: some View {
    VStack(alignment: .leading, spacing: 0) {
      Text("About Us").font(.system(size: 50)).foregroundColor(Color(red: 1, green: 1, blue: 1, opacity: 1)).multilineTextAlignment(.center).frame(maxWidth: .infinity, alignment: .center)
      Image("rsz_ow").resizable().scaledToFit().background(Color.clear).frame(width: 200.0, height: 200.0).padding(.top, 4.0).frame(maxWidth: .infinity, alignment: .center)
      Image("sah").resizable().scaledToFit().background(Color.clear).frame(width: 400.0, height: 100.0).padding(.top, 4.0)
      Text("v 0.0.1").foregroundColor(Color(red: 0.9411764705882353, green: 0.5019607843137255, blue: 0.5019607843137255, opacity: 1)).multilineTextAlignment(.center).frame(width: 372.0).frame(maxWidth: .infinity, alignment: .trailing)
      Text("Made with ♥ for India, in India. ").font(.system(size: 20)).foregroundColor(Color(red: 1, green: 1, blue: 1, opacity: 1)).padding(.top, 20.0).frame(maxWidth: .infinity, alignment: .center)
      Text("http://neophytes.github.io/microsoft-pragyan-hackathon/").padding(.top, 32.0).frame(maxWidth: .infinity, alignment: .center)
    }
    .background(Image("rsz_home_bg").resizable().scaledToFill())
    .frame(maxWidth: .infinity, maxHeight: .infinity)
  }
}

struct About_us_Previews: PreviewProvider {
  static var previews: some View {
    About_us()
  }
}
